docs/content/any/project/changelogs/NEXT.md

```diff
@@ -15,13 +15,44 @@ draft: true
 <!-- TODO: remove warning and replace with "None" if no breaking changes. -->
 
 {{% callout "Warning" "orange" %}}
-  This release contains breaking changes. Be sure to follow migration steps
-  before upgrading.
+This release contains breaking changes. Be sure to follow migration steps
+before upgrading.
 {{% /callout %}}
 
-#### Breaking change 1
+We've made some updates to the syntax of the `resource` predicate, used to configure resources + roles for sqlalchemy-oso-preview. The goal of these changes is to improve the readability of the configuration and make the roles
+features more intuitive to use.
 
-Summary of breaking change.
+#### Rename "perms" -> "permissions" in resource roles configuration
+
+The `roles` parameter of the `resource` previously included a field called `perms` to specify the role permissions.
+We have renamed this field to `permissions` for clarity.
+
+#### Add namespaces to role names
+
+Previously, we required role names to be globally unique. Now, role names will be internally namespaced, removing the globally unique requirement. Like permissions, the role namespace is the resource name specified in the `resource` predicate. Roles names must be unique within a single resource namespace. Roles associated with other resources must be referenced using the namespace. Roles within the same resource can be referenced without the namespace.
+
+#### Remove the `_type:` syntax from the `resource` predicate
+
+The `resource` predicate used to take the resource type parameter as a specializer. Now the predicate takes this parameter as a direct argument, removing the need for the `resource(_type: ResourceType, ...)` syntax. Now, the syntax is `resource(ResourceType, ...)`.
+
+Below is an example `resource` predicate that reflects the above changes. For a more in-depth example, see our [documentation](TODO!!!!) and [example repository](TODO!!!).
+
+```polar
+    # Removed `_type: ` from first argument
+    resource(Organization, "org", actions, roles) if
```

Comment:
Did you consider using the resource type as the namespace instead of requiring folks to pass in a string as the second arg? E.g., `"Organization:member"` instead of `"org:member"` and `resource(Organization, actions, roles) if` instead of `resource(Organization, "org", actions, roles) if`?

Reply:
Yeah I actually did suggest this as another potential improvement: https://www.notion.so/osohq/Reorienting-the-quickstart-and-product-around-the-Oso-Config-f6ca52da55d54cb38c865bac219febc6#ff8f371a0b8e42a5889b8f6d1dab1e1a

We decided not to do it to reduce based on the benefit vs. time to implement. I'm starting to think it might make more sense though, but I think we should take it as a follow-up if we have time. Especially since we still need to implement the config across all languages, which is way higher priority so I'd rather get this done and help with that.

Reply:
okay sounds good 👍